components/financial_assistance/db/seedfiles/translations/en/me/financial_assistance.rb

```diff
@@ -145,6 +145,41 @@
   "en.faa.immigration_status_changed" => "Has this person’s immigration status changed since they were not found eligible for MaineCare (Medicaid) or Cub Care (Children’s Health Insurance Program)",
   "en.faa.has_dependent_with_coverage" => "Did this person have coverage through a job (for example, a parent’s job) that ended in the last 3 months?*",
   "en.faa.dependent_job_end_on" => "What was the last day this person had coverage through the job?*",
+  "en.faa.health_coverage.title" => "Health Coverage for %{applicant}",
+  "en.faa.health_coverage.coverage_title" => "Health Coverage %{number}",
+  "en.faa.health_coverage.desc" => "Tell us about other health coverage options for this person. When you're finished, select 'Continue to Next Step'.",
+  "en.faa.health_coverage.coverage_period_question" => "When was the coverage period for marketplace coverage?",
+  "en.faa.health_coverage.remove_income_type" => "Remove Income Type",
+  "en.faa.health_coverage.add_cov_type" => "Add Additional Health Coverage Type",
+  "en.faa.health_coverage.health_cov_type" => "Other Health Coverage Type",
+  "en.faa.health_coverage.edit" => "Edit",
```

Comment:
do edit delete and save need to have their own keys for health coverage? surely they are already defined elsewhere...

Reply:
They don't need to, I'm leaving for now